src/lib/parameters/param_translation.cpp

```diff
@@ -91,6 +92,18 @@ bool param_modify_on_import(bson_node_t node)
 		}
 	}
 
+	// 2021-08-27: translate LED_RGB_MAXBRT (0-15) to SYS_RGB_MAXBRT(0.f-1.f)
+	if (node->type == BSON_INT32) {
+		if (strcmp("LED_RGB_MAXBRT", node->name) == 0) {
+			// convert integer (0-15) to float percentage
+			node->d = math::constrain(static_cast<float>(node->i) / 15.f, 0.f, 1.f);
```

Comment:
`node->d = math::constrain(static_cast<double>(node->i) / 15.0, 0.0, 1.0);`
casting to double and using double for the templated type should fix the CI failures.  Otherwise everything in this PR looks good to me.  Nice work!

Reply:
Thanks Mark.